dist/PublicLab.Editor.js

```diff
@@ -21945,19 +21945,126 @@ module.exports = function initAutoCenter(_module, wysiwyg) {
 
   $('.wk-commands .woofmark-command-autocenter').click(function() {
     wysiwyg.runCommand(function(chunks,mode) {
-      if(mode == "wysiwyg") { //first convert then replace
-        chunks.selection = ("->"+chunks.selection+"<-")
-        var openingTag = /->/g;
-        var closingTag = /<-/g;
-        chunks.selection = chunks.selection.replace(openingTag,"<center>")
-        chunks.selection = chunks.selection.replace(closingTag,"</center>")
+      if(mode == "wysiwyg") {
```

Comment:
If you could fix formatting and have a **consistent** style, that would be great...